custom-shader/test/index.js

```diff
@@ -4,47 +4,45 @@ import {WasdControlsComponent} from '@wonderlandengine/components';
 /* wle:auto-imports:end */
 
 import {loadRuntime} from '@wonderlandengine/api';
+import {runScreenshotTest} from '../../test-utils.js';
 
 /* wle:auto-constants:start */
-const RuntimeOptions = {
-    physx: false,
-    loader: false,
-    xrFramebufferScaleFactor: 1,
-    canvas: 'canvas',
-};
 const Constants = {
     ProjectName: 'CustomShader',
     RuntimeBaseName: 'WonderlandRuntime',
     WebXRRequiredFeatures: ['local',],
     WebXROptionalFeatures: ['local','hand-tracking','hit-test',],
 };
+const RuntimeOptions = {
+    physx: false,
+    loader: false,
+    xrFramebufferScaleFactor: 1,
+    xrOfferSession: {
+        mode: 'auto',
+        features: Constants.WebXRRequiredFeatures,
+        optionalFeatures: Constants.WebXROptionalFeatures,
+    },
+    canvas: 'canvas',
+};
 /* wle:auto-constants:end */
 
+RuntimeOptions.threads = false; /* Used to run tests in multiple browsers */
```

Comment:
Which browsers don't support the threaded runtime? Or is this just for easier server setup?

Reply:
It's for easier server setup/CI runs on the main repo.

Reply:
They almost never work in Safari as well AFAIK, or our detection fails there